.install-osx.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+brew update
+brew install fish nvm
+mkdir ~/.nvm
+curl -L https://github.com/oh-my-fish/oh-my-fish/raw/master/bin/install | fish
+fish -c 'omf install foreign-env'
+mkdir -p $HOME/.local/share/omf/pkg/nvm && cp -R * $HOME/.local/share/omf/pkg/nvm
+fish -c "nvm install v4.2.4; and nvm alias default v4.2.4"
```

Comment:
With nvm installed by brew, if there is no default node installed, nvm doesn't set NVM_BIN.
I will try the standalone nvm installer on Mac.